packages/devtools_app/lib/src/service/service.dart

```diff
@@ -23,8 +23,8 @@ Future<VmServiceWrapper> _connectWithSse(
       ? uri.replace(scheme: 'http')
       : uri.replace(scheme: 'https');
   final client = SseClient('$uri');
-  final Stream<String>? stream =
-      client.stream?.asBroadcastStream() as Stream<String>?;
+  final Stream<String> stream =
+      client.stream!.asBroadcastStream() as Stream<String>;
```

Comment:
I don't think we should be unconditionally unwrapping this here. On desktop targets, SseClient will come from _fake_sse.dart, which returns a null value for `stream`

Reply:
hm

How do we want to handle this? 
Should the constructor VmServiceWrapper.fromNewVmService  accept null for stream? If yes, what should it pass to the constructor VmService, that requires non-nullable stream.

Reply:
I see. I just verified - looks like we hit `_connectWithWebSocket` on Desktop, not `_connectWithSse`, so what you have should be fine.